config/locales/refresh_companies_house_name.en.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  refresh_companies_house_name:
+    messages:
+      success: You have successfully refreshed the Companies House information.
+      failure: There was an error refreshing the Companies House information. Try again later
```

Comment:
Add a full stop at the end.

Reply:
The wireframe doesn't include one, still want me to add it?

Reply:
Best check with Will.